run/image-processing/app.js

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019 Google LLC. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const app = express();
+
+app.use(bodyParser.json());
```

Comment:
I would include the `require`s inside the region tag.

Reply:
If you only want to show `post('/')` in the sample, that's fine. But since you do full error handling in that function, it seems that you can also show the `require`s.